contracts/script/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "42161": {
+        "GHO_TOKEN": "0x7dfF72693f6A4149b17e7C6314655f6A9F7c8B33",
+        "OWNER": "0xFF1137243698CaA18EE364Cc966CF0e02A4e6327",
```

Comment:
worth renaming to `PROXY_ADMIN_OWNER`?

Reply:
rn also used as the owner the token pool, can decouple both

Reply:
yeah, better